src/classmentors/components/cohort/cohort.js

```diff
@@ -0,0 +1,25 @@
+// import module from 'classmentors/module.js';
+import cohortTmpl from './cohort-view.html!text';
+// import './cohort.css!';
+
+// This here didn't work, not sure why - Amos
+export const component = {
+    cohortTmpl
```

Comment:
you are using a shorthand property names (the property name will be set to the variable name):
```js
const cohortTmpl = 'something';
const component = {cohortTmpl};

console.log({cohortTmpl});
// Object {cohortTmpl: "something"}
```

You should either create the component with `const component = {template: cohortTmpl}` or import the template to a variable name "template":
```
import template from './cohort-view.html!text';

export const component = {template};
```